dhis-2/dhis-web-api/src/main/resources/openapi/EventsExportController.md

```diff
@@ -25,58 +25,87 @@ NOTE: this query parameter has no effect on a response in CSV!
 
 ### `getEventDataValueFile`
 
-Get an event data value file or image for given event and data element UID. Images are returned in
-their original dimension.
+Get an event data value file or image for specified event and data element UID. Images are 
+returned in  their original dimension.
 
 ### `getEventDataValueImage`
 
-Get an event data value image for given event and data element UID. Images are returned in their
+Get an event data value image for specified event and data element UID. Images are returned in their
 original dimension by default. This endpoint is only supported for data elements of value type
 image.
 
 ### `getEventChangeLogsByUid`
 
-Get the change logs of all data elements related to that particular event UID.
+Get the change logs of all data elements belonging to the specified event UID.
 
 ## Common for all endpoints
 
 ### `*.parameter.EventRequestParams.program`
 
+Get events from the specified program.
+
 ### `*.parameter.EventRequestParams.programStage`
 
+Get events from the specified program stage.
+
 ### `*.parameter.EventRequestParams.enrollmentStatus`
 
-Get events from an enrollment in the given status.
+Get events from enrollments that have the specified enrollment status.
 
 ### `*.parameter.EventRequestParams.programStatus`
 
-Get events from an enrollment in the given status.
+Get events from enrollments that have the specified enrollment status.
 
 **DEPRECATED as of 2.42:** Use parameter `enrollmentStatus` instead.
 
 See `enrollmentStatus` for details.
 
 ### `*.parameter.EventRequestParams.followUp`
 
-Get events with given follow-up status of the instance for the given program.
+Get events with specified follow-up status. When set to true, only events marked for
+follow-up will be returned. When set to false, only events not marked for follow-up will be
+returned.
 
 ### `*.parameter.EventRequestParams.trackedEntity`
 
-Get events of tracked entity with given UID.
+Get events belonging to the tracked entity with the specified UID.
 
 ### `*.parameter.EventRequestParams.orgUnit`
 
-Get events owned by given `orgUnit`.
+Get events belonging to the specified `orgUnit`.
```

Comment:
`orgUnit` and `orgUnitMode` should work the same as enrollments?
If yes, also the description should be the same.

Reply:
I think there is slight difference here. In enrollment we support `orgUnits` (multiple) in events we only support `orgUnit` Correct?
I'll sync the description anyway.